src/views/ForgotPasswordPageChange/index.js

```diff
@@ -0,0 +1,76 @@
+import React, { Component } from "react";
+import Form from "react-bootstrap/Form";
+import Button from "react-bootstrap/Button";
+import { Link, Route } from "react-router-dom";
+import LoginPage from "../LoginPage";
+
+var requirements = {
+  password: undefined,
+  confirmpassword: undefined
+};
+
+class ForgotPasswordPageChange extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      code: undefined
+    };
+  }
+
+  handleChange = event => {
+    const { info, value } = event.target;
+
+    this.setState({
+      [info]: value
+    });
+    console.log(this.state);
+  };
+
+  handleSubmit(event) {
+    event.preventDefault();
+    requirements.password = this.state.code;
```

Comment:
then here there's no need to set the requirement object here! this is where we'll call things like AWS.Cognito.changePassword(this.state.code, this.state.confirmPassword) (or whatever) directly using the state with no need for requirements